src/index.html

```diff
@@ -11,7 +11,195 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset class="fieldset">
+        <legend>Personal information</legend>
+
+        <label class="form-field" for="Surname">
+
+          Surname:
+          <input
+            type="text"
+            name="Surname"
```

Comment:
check out point 4 in the [checklist](https://github.com/mate-academy/layout_html-form/blob/master/checklist.md). `name` attribute should use camelCase, fix it everywhere
```suggestion
            name="surname"
```